libs/nuxt/src/webpack.ts

```diff
@@ -3,7 +3,7 @@ import TsconfigPathsPlugin from 'tsconfig-paths-webpack-plugin';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function modifyTypescriptAliases(config: any, projectRoot: Path): void {
-  ['~~', '@@', '~', '@', 'assets', 'static'].forEach(
+  ['~~', '@@', '@', 'assets', 'static'].forEach(
```

Comment:
This fixes the compilation error, but like I mentioned [here](https://github.com/ZachJW34/nx-plus/issues/125#issuecomment-731704018) please remove all of the aliases.